playwright.config.ts

```diff
@@ -29,7 +33,12 @@ export default defineConfig({
     // baseURL: 'http://localhost:3000',
```

Comment:
```suggestion
    baseURL: 'https://practicesoftwaretesting.com',
```
See below comments to learn why this change is beneficial